web/Views/Home/Privacy.cshtml

```diff
@@ -1,4 +1,4 @@
-﻿@{
+﻿{

```

Comment:
I think this won't workk, you'll get this printed

Reply:
My mistake, I forgot to re-add the @